net-misc/networkmanager-fortisslvpn/files/networkmanager-fortisslvpn-1.2.2-location-fix.diff

```diff
@@ -0,0 +1,11 @@
+diff --git a/src/Makefile.am b/src/Makefile.am
```

Comment:
If the patch comes from upstream, please use `git format-patch` so we get all information from the commit. If the patch does not apply cleanly, it is recommended to rebase it on the target version for the tree.